shell/edit/auth/oidc.vue

```diff
@@ -59,42 +68,74 @@ export default {
       };
     },
 
+    validationPassed() {
+      if ( this.model.enabled && !this.editConfig ) {
+        return true;
+      }
+
+      const { clientId, clientSecret } = this.model;
+      const isValidScope = this.model.id === 'keycloakoidc' || this.oidcScope?.includes('openid');
```

Comment:
This could be optimised / clearer by exiting early if this condition is false (as it's in both return statements below). Doesn't need changing now though